helm/istio/charts/tracing/templates/jaeger-streaming-elasticsearch.yaml

```diff
@@ -0,0 +1,114 @@
+{{ if and (eq .Values.provider "jaeger") (eq .Values.jaeger.template "streaming-elasticsearch") }}
+apiVersion: jaegertracing.io/v1
+kind: "Jaeger"
+metadata:
+  name: "jaeger"
+  namespace: {{ .Release.Namespace }}
+  labels:
+    chart: {{ template "tracing.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+spec:
+  strategy: streaming
+
+  query:
+    {{- if and .Values.jaeger.hub .Values.jaeger.tag .Values.jaeger.queryImage }}
+    image: {{ .Values.jaeger.hub }}/{{ .Values.jaeger.queryImage }}:{{ .Values.jaeger.tag }}
+    {{- end }}
+    options:
+      query:
+        base-path: /
+    annotations:
+      {{- range $key, $value := .Values.jaeger.annotations }}
+        {{ $key }}: {{ $value | quote }}
+      {{- end }}
+
+  collector:
+    {{- if and .Values.jaeger.hub .Values.jaeger.tag .Values.jaeger.collectorImage }}
+    image: {{ .Values.jaeger.hub }}/{{ .Values.jaeger.collectorImage }}:{{ .Values.jaeger.tag }}
+    {{- end }}
+    annotations:
+      {{- range $key, $value := .Values.jaeger.annotations }}
+        {{ $key }}: {{ $value | quote }}
+      {{- end }}
+
+  ingester:
+    {{- if and .Values.jaeger.hub .Values.jaeger.tag .Values.jaeger.ingesterImage }}
+    image: {{ .Values.jaeger.hub }}/{{ .Values.jaeger.ingesterImage }}:{{ .Values.jaeger.tag }}
+    {{- end }}
+    annotations:
```

Comment:
Not quite sure how the template works, but don't you have to specify the broker URLs?

Reply:
No, as this will use the self provisioned approach.

Reply:
Sorry, I misread your initial comment and thought you were avoiding the auto provisioning feature. LGMT then